libby/action/ActionList.lib.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { describe, add } from '@sparkpost/libby-react';
-import { ActionList, Button, Inline, Popover, Panel, Box } from '@sparkpost/matchbox';
+import { Button, Inline, Popover, Panel, Box } from '@sparkpost/matchbox';
+import { ActionList } from '../../packages/matchbox/src';
```

Comment:
I couldn't figure out how to get types imported through the webpack alias. might need to convert all stories to this import structure to get type checking

Reply:
I'm pretty sure you have to set the webpack alias and then set a matching url in a `paths` object in tsconfig.json like what we do here https://github.com/SparkPost/design.sparkpost.com/blob/main/tsconfig.json#L17